web/themes/custom/cloudy/src/components/_skip-link.scss

```diff
@@ -8,7 +8,7 @@
     -webkit-transform: translateX(-50%);
     -ms-transform: translateX(-50%);
     transform: translateX(-50%);
-    z-index: 50;
+    z-index: 1;
```

Comment:
I'm curious why this skip link, which is visually hidden, has all of these styles to alter it's positioning, appearance, and z-index (none of which seem to matter since it's visually hidden...) This isn't really a blocker, just something I noticed that could potentially be removed if unnecessary to reduce our CSS size.

Reply:
I had the same thought. The skip link does appear when focused on by tabbing to it or using other assistive technology. Therefore, we need to provide styling – OR we could remove our custom skip link style and instead use Drupal core skip link styling.

I wonder if our current styling is problematic as the skip link overlaps our alert text (when present), which could impact readability/intrusion on the visual design. @julegan did you want to provide any further thoughts/recommendations on our current usage?

Resources;
http://web-accessibility.carnegiemuseums.org/code/skip-link/
https://webaim.org/techniques/skipnav/

Our custom skip link style:
<img width="1577" alt="Screen Shot 2021-03-01 at 10 21 34 AM" src="https://user-images.githubusercontent.com/30271981/109540872-ee248600-7a77-11eb-830b-432906e0bcf5.png">


Drupal core skip link style:
<img width="1042" alt="Screen Shot 2021-03-01 at 10 22 28 AM" src="https://user-images.githubusercontent.com/30271981/109540955-08f6fa80-7a78-11eb-935a-f9316a1ae5f9.png">


City of Boston skip link:
<img width="1132" alt="Screen Shot 2021-03-01 at 10 24 53 AM" src="https://user-images.githubusercontent.com/30271981/109541185-5ffccf80-7a78-11eb-9ec1-f0a077c71909.png">

Reply:
Ah, that makes sense. I guess the code would be more self-explanatory if those styles were applied to the :focus pseudo selector instead of the main element though. But it works either way.